### DISCUSSION TITLE
Is anyone trying this for Essential Tremor?

### DISCUSSION THREAD

**Discussion by ClaudeJGreengrass** [kriswilk/buzzah#14] [Q&A]
I have Essential Tremor and have tried electrical neurostimulation of the Median and Radial nerve in the wrist for the treatment of ET and it works for me.  Most the the research I have read regarding vibration stimulation is for the treatment of Parkinson's.  Anyone with ET looking into vibration stimulation treatment?  Any external references would also be welcome.  thx

**Comment by TactileDesign** [comment]
Hello Claude,  There are very few people on this site currently and it looks like no one has any experience with essential tremor.  I can say though that I saw a video on Youtube where Dr Tass was posed with this same question. (I cant remember which interview it was - but in one of the Q&A sessions at the end)  His response as I recall was that, while he had not yet treated anyone with ET, he felt that there was a strong chance that the gloves could be helpful for ET.  It is one of the many things on his long list of to do items in the coming years.  That said, there is nothing stopping YOU from becoming the first patient!  I can tell you that this Buzzah design is the closest I have seen to the original Tass glove with the added bonus that it is easy to implement!   If you decide to give it a try, please report back to us and let us know what kind of results you experience.

**Reply by ClaudeJGreengrass** [reply]
I'm considering it, but I still have work to do on my electrical neurological stimulation device. (the portable one). 

For the record, since I posted the original query, I have  become aware of at least 4 groups, commercial and academic, that are working on potential vibration treatments for Essential Tremor.
